be/src/vec/core/field.h

```diff
@@ -936,10 +936,6 @@ struct Field::EnumToType<Field::Types::UInt64> {
     using Type = UInt64;
 };
 template <>
-struct Field::EnumToType<Field::Types::UInt128> {
```

Comment:
why delete this?

Reply:
This is my mistake, it really shouldn't have been deleted